Symfony/CS/ConfigInterface.php

```diff
@@ -105,4 +105,20 @@ public function usingCache();
      * @return bool
      */
     public function usingLinter();
+
+    /**
+     * Sets the path to the cache file.
+     *
+     * @param string $cacheFile The path to the cache file
+     *
+     * @return ConfigInterface The same instance
+     */
+    public function setCacheFile($cacheFile);
+
+    /**
+     * Returns the path to the cache file.
+     *
+     * @return string The path to the cache file
```

Comment:
@GrahamCampbell 

However, please have a look, for example here:

* https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/1.5/Symfony/CS/ConfigInterface.php#L33

So, actually, all of these annotations have those comments. I wouldn't want them, though, so I'm totally :+1:.